GVFS/GitHooksLoader/GitHooksLoader.cpp

```diff
@@ -135,7 +132,7 @@ int ExecuteHook(const std::wstring &applicationName, wchar_t *hookName, int argc
         NULL,           // Process handle not inheritable
         NULL,           // Thread handle not inheritable
         TRUE,           // Set handle inheritance to TRUE
-        CREATE_NO_WINDOW, // Process creation flags
+        NULL,           // Process creation flags

```

Comment:
I seriously wonder whether this flag is there on purpose, to avoid "flashing windows" when, say, running `git.exe` from Visual Studio.

Reply:
Good point, I'll check that.

Reply:
I observed the updated hook getting called in procmon from VS, but there was no flashing window that I could see. I would expect that VS is already launching its git command with CREATE_NO_WINDOW, so the hooks loader is inheriting that state and passing it on.